lib/SampleService/core/sample.py

```diff
@@ -27,9 +27,9 @@
 _MAX_SAMPLE_NAME_LEN = 256
 _MAX_SAMPLE_NODES = 10000
 
-_META_MAX_SIZE_B = 100000  # based on serialized json
+_META_MAX_SIZE_B = 150000  # based on serialized json
 _META_MAX_KEY_SIZE = 256
-_META_MAX_VALUE_SIZE = 1024
+_META_MAX_VALUE_SIZE = 8192
```

Comment:
I don't remember now, but I thought I set this to 1KB to avoid indexing problems - like if you try to index a field larger than X it causes problems, but I'm not sure. Are you sure this won't cause indexing issues?

Reply:
Shouldn't be problem for Elastic in terms of field size. I think there is a theoretical limit at 32766 bytes for Lucene’s term byte-length limit

Reply:
I was talking about Arango indexing rather than Elastic indexing

Reply:
I don't know about Arango document/value limits. From what I can tell/read it's more linked to total document size rather than limits for individual fields

Reply:
Hmm. I'm just wondering why I made the limit 1k. I assume I had a good reason but I've forgotten. 
I guess if we need to index the metadata - which is certainly possible - all the meta values would have to be indexed, which could potentially blow up the index, and therefore memory usage, if we allow large values.

Note to self - document rationale for limit constants in the future

On second thought - is this a case that happens many time per sample node or just once - like say a description? For that kind of thing maybe it makes sense to keep the metadata fields small and instead add a `description` field to the node or something.